src/vs/workbench/common/theme.ts

```diff
@@ -401,6 +401,24 @@ export const PANEL_SECTION_BORDER = registerColor('panelSection.border', {
 	hcLight: PANEL_BORDER
 }, localize('panelSectionBorder', "Panel section border color used when multiple views are stacked horizontally in the panel. Panels are shown below the editor area and contain views like output and integrated terminal. Panel sections are views nested within the panels."));
 
+// < --- Output Editor -->
+
+const OUTPUT_EDITOR_BACKGROUND = registerColor('outputEditor.background', {
+	dark: null,
+	light: null,
+	hcDark: null,
+	hcLight: null
+}, localize('outputEditorBackground', "Output editor background color."));
```

Comment:
I'm a bit confused, what is this color needed for? And why doesn't it have a default?

Reply:
Yes this color is used as the background of the output editor. The reason it does not have a default is because of comment https://github.com/microsoft/vscode/pull/200251#issuecomment-1850282256. Since the color is null, it defaults to the base color of the panel view.

There is also another color, which is the color of the sticky scroll background. This color defaults to the output editor background.

Reply:
Ah, this is because of the `var` fallback value in the added css.